iothub/service/src/DirectMethod/DirectMethodRequest.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Parameters to execute a direct method on the device
+    /// </summary>
+    public class DirectMethodRequest
+    {
+        /// <summary>
+        /// Method to run
+        /// </summary>
+        [JsonProperty("methodName", Required = Required.Always)]
+        public string MethodName { get; set; }
+
+        /// <summary>
+        /// The timeout (in seconds) before the direct method request will fail if the device doesn't respond to the request.
+        /// This timeout may happen if the target device is slow in handling the direct method.
+        /// </summary>
+        [JsonProperty("responseTimeoutInSeconds", NullValueHandling = NullValueHandling.Ignore)]
+        public int? ResponseTimeout { get; set; }
```

Comment:
These two timeout fields used to be timespans, but I don't feel that represents the field well here. The json object sent over the wire represents this timeout as an int. On top of that, making these timespans suggests a level of precision (1.15 seconds vs 1 second) that isn't possible

Reply:
I'd add the precision info in doc comments.